kiwi/volume_manager/btrfs.py

```diff
@@ -354,19 +357,48 @@ def _xml_pretty(self, toplevel_element):
         return xml_data_domtree.toprettyxml(indent="    ")
 
     def _create_quota_group_info(self):
-        snapper_configs = ''.join([self.mountpoint, '/@/etc/snapper/configs/'])
-        if os.path.exists(snapper_configs):
-            with open(snapper_configs + 'root', 'w') as snapper_root:
-                snapper_root.write(
-                    '{0}{1}'.format(self._get_quota_group_id(), os.linesep)
-                )
+        root_path = os.sep.join([self.mountpoint, '@/.snapshots/1/snapshot'])
+        snapper_default_conf = os.sep.join(
+            [root_path, 'etc/snapper/config-templates/default']
```

Comment:
I think that information should be move into the Defaults space

Reply:
right